openssl/src/symm.rs

```diff
@@ -534,4 +588,42 @@ mod tests {
 
         cipher_test(super::Cipher::des_ecb(), pt, ct, key, iv);
     }
+
+    #[test]
+    fn test_aes128_gcm() {
+        let key = "0e00c76561d2bd9b40c3c15427e2b08f";
```

Comment:
Why not format the test vectors as `[u8]`? The tests would be a bit more straightforward not converting to/from hex everywhere.